geps/gep-1494/index.md

```diff
@@ -0,0 +1,59 @@
+# GEP-1494: Auth in Gateway API
+
+* Issue: [#1494](https://github.com/kubernetes-sigs/gateway-api/issues/1494)
+* Status: Provisional
+
+(See status definitions [here](/geps/overview/#gep-states).)
+
+## TLDR
+
+Allow authentication works as a policy that can  be attached to both the Gateway and HTTPRoute.
+
+## Goals
+
+1). In the default policy, it can add an ExtensionFilter that specifies the auth so that Gateway can be attached to.
+2). Individual HTTPRoutes can choose to opt out by specifying an empty extension, or a null extension. 
+3). It can also have disabled authentication.
+
+
+## Non-Goals
+
+1). How authentication is implemented.
```

Comment:
I think that the implementation of the authentication will eventually be required in this GEP, so it isn't really correct to put this in here.

Maybe, let's add a `## Deferred Goals` section for now, to mark goals that we'll get to later in this process. (I think I'll open a PR to the GEP template to add this as well).